src/main.ts

```diff
@@ -172,14 +176,13 @@ export function wrap<T>(
       const defaultContext = _makeDefaultContext(cloudFunction, options, data);
 
       if (
-        has(defaultContext, 'eventType') &&
-        defaultContext.eventType !== undefined &&
+        !!defaultContext.eventType &&
```

Comment:
FWIW we don't typically do bool casting. Don't need to remove it though.

Reply:
Removed bool casting to match existing code convention.